MetaMask/metamask-extension
# Fix prompt for backing the SRP

## Explanation
Fixed prompt for backing the SRP.


## More Information
* Fixes #13277


## Screenshots/Screencaps


### Before

https://user-images.githubusercontent.com/92310504/198003736-91c95f3c-1d7a-4a0f-a956-96e7a741f89b.mov




### With Ganache

https://user-images.githubusercontent.com/92310504/198003842-ccd8a14d-3700-4054-b1d8-b6df045f9718.mov

### After


https://user-images.githubusercontent.com/92310504/198003991-4a157b67-f2dd-49f6-8d8a-412ddbaa94b5.mov

### With Ganache

https://user-images.githubusercontent.com/92310504/198004055-5cddde83-979b-4dab-b135-695df0035695.mov




## Manual Testing Steps

How to reproduce it (all these options are with `Remind me later to backup SRP` option while creating a wallet):

1. Create a new wallet, switch to Sepolia test network (account with 0 balance), send some amount from another wallet and warning will not appear like before.
2. Switch to Mainnet, import a token (for example UST) and warning will not appear like before.
3. Connect a hardware wallet, import a hardware account and import a token (for example UST) and warning will not appear like before.
4. Test the Mainnet functionality without having to actually use the Mainnet by using the Ganache's forking feature (https://github.com/MetaMask/metamask-extension/pull/15944/files).
